### PR TITLE
Include Significant Mutational Signatures Component in Patient View Header

### PR DESCRIPTION
**Pull request by jxu8**
New component is based on pull request for GDD: 
https://github.com/cBioPortal/cbioportal-frontend/pull/724

### CODE REVIEW

File: src/pages/patientView/PatientViewPage.tsx [hunk 1]
```diff
@@ -247,6 +251,12 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                                 )
                             }
                         </span>
+
+                        {patientViewPageStore.clinicalDataGroupedBySample.isComplete && patientViewPageStore.mutationalSignatureData.isComplete
```
Comment (adamabeshouse) [inline src/pages/patientView/PatientViewPage.tsx]: maybe a LoadingIndicator component could be useful here?

File: src/pages/patientView/patientHeader/SignificantMutationalSignatures.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                        <td style={{paddingTop: 3}}>{this.progressBar(significantMutationalSignaturesForSample.significantSignatures[significantSignatureValue].toString(), false)}</td>
+                        </tr>)}
+                    </table>
+                }
+
+            </div>
+        );
+    }
+
+    public progressBar(confidence:string, determineColor:boolean) {
```
Comment (adamabeshouse) [inline src/pages/patientView/patientHeader/SignificantMutationalSignatures.tsx]: this doesnt need anything in the class right? like it doesn't reference `this`. so I dont think it needs to be an instance method. you can move it out as a helper method, maybe into a file called `MutationalSignaturesUtils.tsx`

File: src/pages/patientView/patientHeader/SignificantMutationalSignatures.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                    textAlign:"center"}}
+                >{confidencePerc}%</div>
+            </div>
+        );
+    }
+
+}
+
+export function prepareMutationalSignaturesForHeader(mutationalSignatureData: IMutationalSignature[], mutationalSignatureMetaData: IMutationalSignatureMeta[],
+                                                     uniqueSampleKey: string): ISignificantMutationalSignaturesForSample{
```
Comment (adamabeshouse) [inline src/pages/patientView/patientHeader/SignificantMutationalSignatures.tsx]: the way we've been organizing code in the frontend is to move stuff like this into `MutationalSignaturesUtils`

Comment (jxu8) [reply src/pages/patientView/patientHeader/SignificantMutationalSignatures.tsx]: Ok, sounds good! I can move the equivalent function for the mutational signature table in there as well for organization.

**Pull request merged**